scipy/stats/tests/test_axis_nan_policy.py

```diff
@@ -710,7 +711,7 @@ def small_sample_generator(n_dims):
                 # After broadcasting, all arrays are the same shape, so
                 # the shape of the output should be the same as a single-
                 # sample statistic. Use np.mean as a reference.
-                concat = stats._stats_py._broadcast_concatenate(samples, axis)
+                concat = stats._stats_py._broadcast_concatenate(samples, axis, paired=paired)
```

Comment:
This test catches regressions when the broadcasted shape is size-0. The bug wasn't caught before because the test neglected `paired`, too.